### PR TITLE
Augmentations now subclass Transform

### PR DESCRIPTION
**Pull request by scart97**

### CODE REVIEW

**Comment by scart97** [comment]
<p>The only one i know is <code>torch.nn.utils.rnn.pad_sequence</code>, but it only does constant padding at the back, and probably will not work with our data because RNN's use a different ordering for batch size vs sequence length </p>

---

 <a href='https://app.reviewnb.com/rbracco/fastai2_audio/pull/41/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>
